src/components/FeedbackBox/FeedbackBox.tsx

```diff
@@ -0,0 +1,441 @@
+  /** Toggles the category radio group field. */
+  showCategoryField?: boolean;
+  /** Toggles the email input field. */
+  showEmailField?: boolean;
+  /** Used to populate the label on the open button and the `Drawer`'s
+   * header title. */
+  title: string;
+  /** Used to specify what screen should be displayed. */
+  view?: ViewType;
+}
+
+/**
+ * The `FeedbackBox` component renders a fixed-positioned button on the bottom
+ * right corner of a page that opens a Chakra `Drawer` popup component. Inside
+ * of the popup, a form is rendered with fields that allows users to provide
+ * feedback. The `FeedbackBox` component does *not* call any API with the
+ * submitted data; that feature is the responsibility of the consuming
+ * application.
+ */
+export const FeedbackBox = chakra(
+  forwardRef<any, FeedbackBoxProps>(
+    (
+      {
+        className,
+        confirmationText,
+        descriptionText,
+        hiddenFields,
+        id = "feedbackbox",
+        isInvalidComment = false,
+        isInvalidEmail = false,
+        notificationText,
+        onSubmit,
+        showCategoryField = false,
+        showEmailField = false,
+        title,
+        view = "form",
+        isOpen,
+        onOpen,
+        onClose,
+        ...rest
+      },
+      ref?
+    ) => {
+      // We want to keep internal state for the view but also
+      // update if the consuming app updates it, based on API
+      // success and failure responses.
+      const [viewType, setViewType] = useStateWithDependencies(view);
+      const [isSubmitted, setIsSubmitted] = useState<boolean>(false);
+      // Helps keep track of form field state values.
+      const { state, setCategory, setComment, setEmail, clearValues } =
+        useFeedbackBoxReducer();
+      // Chakra's hook to control Drawer's actions.
+      const disclosure = useDisclosure();
+      const finalIsOpen = isOpen ? isOpen : disclosure.isOpen;
+      const finalOnOpen = onOpen ? onOpen : disclosure.onOpen;
+      const finalOnClose = onClose ? onClose : disclosure.onClose;
+      const focusRef = useRef<HTMLDivElement>();
+      const styles = useMultiStyleConfig("FeedbackBox", {});
+      const isFormView = viewType === "form";
+      const isConfirmationView = viewType === "confirmation";
+      const isErrorView = viewType === "error";
+      const confirmationTimeout = 3000;
+      const maxCommentCharacters = 500;
+      const closeAndResetForm = () => {
+        finalOnClose();
+        setViewType("form");
+        clearValues();
+      };
+      const internalOnSubmit = (e) => {
+        e.preventDefault();
+        let submittedValues = { ...state };
+        if (hiddenFields) {
+          submittedValues = { ...submittedValues, ...hiddenFields };
+        }
+        onSubmit && onSubmit(submittedValues);
+        setIsSubmitted(true);
+      };
+      const notificationElement =
+        isFormView && notificationText ? (
+          <Notification
```

Comment:
The top/bottom padding within the `Notification` needs to be reduced to 8px.  Refer to the Figma designs.